crates/sui-benchmark/src/bin/stress.rs

```diff
@@ -0,0 +1,543 @@
+                let mut min_latency: Duration = Duration::MAX;
+                let mut max_latency: Duration = Duration::ZERO;
+                let mut num_in_flight: u64 = 0;
+                let mut num_submitted: u64 = 0;
+                let mut num_no_gas = 0;
+                for (_, v) in stat_collection.iter() {
+                    total_qps += v.num_success as f32 / duration.as_secs() as f32;
+                    num_success += v.num_success;
+                    num_error += v.num_error;
+                    num_no_gas += v.num_no_gas;
+                    num_submitted += v.num_submitted;
+                    num_in_flight += v.num_in_flight;
+                    min_latency = if v.min_latency < min_latency {
+                        v.min_latency
+                    } else {
+                        min_latency
+                    };
+                    max_latency = if v.max_latency > max_latency {
+                        v.max_latency
+                    } else {
+                        max_latency
+                    };
+                }
+                let denom = num_success + num_error;
+                let _error_rate = if denom > 0 {
+                    num_error as f32 / denom as f32
+                } else {
+                    0.0
+                };
+                write(format!("Throughput = {}, min_latency_ms = {}, max_latency_ms = {}, num_success = {}, num_error = {}, no_gas = {}, submitted = {}, in_flight = {}", total_qps, min_latency.as_millis(), max_latency.as_millis(), num_success, num_error, num_no_gas, num_submitted, num_in_flight));
+            }
+        }));
+    let _: Vec<_> = try_join_all(tasks).await.unwrap().into_iter().collect();
+}
+
+#[tokio::main]
+async fn main() {
+    let env_filter = EnvFilter::try_from_default_env().unwrap_or_else(|_| EnvFilter::new("warn"));
+    let subscriber_builder =
+        tracing_subscriber::fmt::Subscriber::builder().with_env_filter(env_filter);
+    let subscriber = subscriber_builder.with_writer(std::io::stderr).finish();
+    set_global_default(subscriber).expect("Failed to set subscriber");
+    let opts: Opts = Opts::parse();
+
+    // This is the maximum number of increment counter ops in flight
+    let max_in_flight_ops = opts.target_qps as usize * opts.in_flight_ratio;
+
+    let configs = test_and_configure_authority_configs(opts.committee_size);
+
+    // initialize the right kind of benchmark
+    let (payload, addresses, clients) = if let TransactionType::SharedCounter =
+        opts.transaction_type
+    {
+        init_shared_counter_benchmark(max_in_flight_ops, configs).await
+    } else if let TransactionType::TransferObject = opts.transaction_type {
+        init_object_transfer_benchmark(max_in_flight_ops, opts.num_transfer_accounts, configs).await
+    } else {
+        unreachable!()
+    };
+    run(clients, addresses, payload, opts).await
+}
+
+fn write(str: String) {
+    eprintln!("{}", str);
+}
```

Comment:
remove this function and either use `eprintln` directly or maybe use tracing instrumentation instead, eg `info`